ui/src/components/Content/ContentCard.tsx

```diff
@@ -30,19 +28,20 @@ export default function ContentCard({
       <div
         className={classNames(title ? 'shadow-slate-950 px-5 py-4 shadow-lg relative z-30' : '')}
       >
-        {title ? (
-          <div className="mb-5">
-            <h1 className=" text-lg text-slate-50">{title}</h1>
-            <span className="text-2xs mt-1 block">
-              <Time date={date} />
-            </span>
-          </div>
-        ) : null}
-
-        <div className="flex items-center justify-between">
-          {button && button}
-          <span className="text-3xs leading-none">{[idPrefix, id].filter(Boolean).join(': ')}</span>
+        <div className="flex items-center justify-between leading-7">
+          {title ? (
+            <div className="flex items-center gap-2">
+              {type === 'event' ? (
```

Comment:
Non-blocking nit: I find ternaries hard to read when they're in JSX. But maybe that's just me so I'll still accept

Reply:
Sometimes it's easier to write, but in this case is not even necessary, I changed it to something more readable 😄